modeling.py

```diff
@@ -0,0 +1,146 @@
+import nimfa
+import pandas as pd
+from nfldb_tables import NFLFrames
+from scipy.sparse import csr_matrix as csrm
+
+
+class PositionNMF(object):
```

Comment:
Ok.  I can wait until that's up.  Will that be on this PR or on a different branch?